static/app/views/organizationIntegrations/integrationMainSettings.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import {t} from 'app/locale';
+import {Integration, Organization} from 'app/types';
+import Form from 'app/views/settings/components/forms/form';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import {Field} from 'app/views/settings/components/forms/type';
+
+type Props = {
+  integration: Integration;
+  organization: Organization;
+  onUpdate: () => void;
+};
+
+type State = {
+  integration: Integration;
+};
+
+class IntegrationMainSettings extends React.Component<Props, State> {
+  state: State = {
+    integration: this.props.integration,
+  };
+
+  handleSubmitSuccess = (data: Integration) => {
+    this.props.onUpdate();
+    this.setState({integration: data});
+  };
+
+  get initialData() {
+    const {integration} = this.props;
+
+    return {
+      name: integration.name,
+      domain: integration.domainName || '',
```

Comment:
This is a little different than the form here https://github.com/getsentry/sentry/pull/25789#issuecomment-830466392 because in the "install" flow we actually just ask for the base but I think either it's fine to just let them edit the full domain name. If I were to update anything, I would change the install form, but I think once they see it like this we can add more info about what this domain would be used for (tbh only stacktrace linking i think)